docs/updates/2019-04.md

```diff
@@ -0,0 +1,197 @@
+# April 2019
+
+Welcome to the April 2019 Qlik Core update. We're highlighting a couple of important changes and improvements which we
+hope you will like.
+
+## Changing engine expiration time
+
+## Enhancements in catwalk
+
+### Demo app
+
+Looking at the visitor statistics we could see that a lot of people opened and left catwalk without opening a Qlik
+application. One possible explanation to this behavior is that these visitors are curious about the tool but lacking
+a running engine or a configured Qlik Sense Enterprise prevented them from further exploring. To mitigate this we have
+added a
+[demo app](https://catwalk.core.qlik.com/?engine_url=wss://apps.core.qlik.com/app/doc/01775889-c700-413f-9b0e-6ba1837c52b0/)
+which can be used as an introduction to the tool and is automatically loaded if no engine URL is specified.
+
+### Default websocket
+
+If you are mainly working with one engine instance, using the new default socket connection feature might be of use.
+By clicking on the star, top right in catwalk, the current websocket will be stored in the browser's local storage and
```

Comment:
remove "using". -> 
"If you are mainly working with one engine instance the new default socket connection feature might be of use."